manifest.md

```diff
@@ -191,6 +191,10 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
         This REQUIRED property is the digest of the content, as defined by the [Descriptor](descriptor.md) digest format.
 
+    - **`urls`** *array*
+
+       For an ordinary layer, this is empty, and the layer contents can be retrieved directly from the registry. For a layer with mediatype of application/vnd.docker.image.rootfs.foreign.diff.tar.gzip, this contains a non-empty list of URLs from which this object can be downloaded.
```

Comment:
you can see `urls` is there https://github.com/opencontainers/image-spec/blob/master/descriptor.md#reserved I just wanted to bring up the discussion here for a future version of the spec probably